src/relationships-README.md

```diff
@@ -0,0 +1,36 @@
+# Incentive Relationships Documentation
```

Comment:
Could you link this from the `### Data` section in CONTRIBUTING?

I also wonder if we should start a `/docs` folder to house these? (Thinking: would there be others? e.g. our style guide for descriptions?)

Reply:
I added the link to CONTRIBUTING.

I agree it would be good to put this in a better place like a docs folder, so I moved it. For now I don't know what else should go in there but I imagine there will be more things soon.

Do you think we should convert the style guide google doc to markdown and put it in there?